index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Setup</title>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="container">
+    <section class="nav-bar">   
+      <a class="logo" href="">My Logo</a> 
+      <img class="union" src="/images/Union.png" alt="">   
+   </section> 
+  <main class="content">
+     <h1>I'm Shadrach<br/>
+    Glad to see<br>
+     you!
+     </h1>
+  </main>
+  <article class="second-text">
+     <p>I'm a software developer! I can help you<br>build a product feature or website. Look<br>
```

Comment:
- [ ] I think it would be a great idea to wrap this section in a `<section>` tag instead of using `<main>` and `<article>`. This is because first, the main tag is used for your project's main content 🗒️, in this case, is a portfolio, so ideally you would want to have your projects as the main content. Second, both, the title and introduction text represents the same section, you could call it a `headline` so a `<section>` tag would make much sense in this case, and would improve how the browser understands your page (semantics) 🖊️. Kindly change this and don't forget to use flexbox for the layout! 🛠️